tpp-app/src/settings/NotificationAccordion.js

```diff
@@ -0,0 +1,354 @@
+    return (
+        <View>
+        <Picker
+         selectedValue={selectedValue}
+         onValueChange={(itemValue) =>
+         {setSelectedValue(itemValue),
+         POSTRemindLogSymptomsFreq(selectedValue)}}>
+        <Picker.Item 
+    label={"Every day"}
+    value={"Every day"} 
+    />
+    <Picker.Item 
+    label={"Every week"}
+    value={"Every week"} 
+    />
+    <Picker.Item 
+    label={"Every month"}
+    value={"Every month"} 
+    />
+    <Picker.Item 
+    label={"Only during period"}
+    value={"Only during period"} 
+    />
+        </Picker>
+    </View>
+    );
+}
+
+const SymptomTimePicker = (props) => {
+    var storedTime = "10:00"
+    var storedMeridian = "AM"
+    if (typeof props.setValue == 'string'){
+        const values = props.setValue.split("")
+        storedTime = values[0]
+        storedMeridian = values[1]
+    }
+
+    const [selectedValue, setSelectedValue] = useState(storedTime);
+    const [selectedMeridian, setMeridian] = useState(storedMeridian);
+return(
+    <View>
+    <Picker  
+    selectedValue={selectedValue}
+    onValueChange={(itemValue) =>
+    setSelectedValue(itemValue)
+    }>
+     <Picker.Item 
+    label={"1:00"}
+    value={"1:00"} 
+    />
+    <Picker.Item 
+    label={"2:00"}
+    value={"2:00"} 
+    />
+    <Picker.Item 
+    label={"3:00"}
+    value={"3:00"} 
+    />
+    <Picker.Item 
+    label={"4:00"}
+    value={"4:00"} 
+    />
+    <Picker.Item 
+    label={"5:00"}
+    value={"5:00"} 
+    />
+    <Picker.Item 
+    label={"6:00"}
+    value={"6:00"} 
+    />
+    <Picker.Item 
+    label={"7:00"}
+    value={"7:00"} 
+    />
+    <Picker.Item 
+    label={"8:00"}
+    value={"8:00"} 
+    />
+    <Picker.Item 
+    label={"9:00"}
+    value={"9:00"} 
+    />
+    <Picker.Item 
+    label={"10:00"}
+    value={"10:00"} 
+    />
+    <Picker.Item 
+    label={"11:00"}
+    value={"11:00"} 
+    />
+    <Picker.Item 
+    label={"12:00"}
+    value={"12:00"} 
+    />
+    </Picker>
+    <Picker  
+            selectedValue={selectedMeridian}
+            onValueChange={(itemValue) =>
+            setMeridian(itemValue)
+            }>
+            <Picker.Item 
+            label={"AM"}
+            value={"AM"} 
+            />
+            <Picker.Item 
+            label={"PM"}
+            value={"PM"} 
+            />
+            </Picker>
+    </View>
+)
+}
+const PeriodTimePicker = (props) => {
+    var storedTime = "10:00"
+    var storedMeridian = "AM"
+    if (typeof props.setValue == 'string'){
+        const values = props.setValue.split("")
+        storedTime = values[0]
+        storedMeridian = values[1]
+    }
+
+    const [selectedValue, setSelectedValue] = useState(storedTime);
+    const [selectedMeridian, setMeridian] = useState(storedMeridian);
+return(
+    <View>
+    <Picker  
+    selectedValue={selectedValue}
+    onValueChange={(itemValue) =>
+    setSelectedValue(itemValue)
+    }>
+     <Picker.Item 
```

Comment:
You can probably rewrite this using `map` and just compute `label` & `value` based off the index